Switcheroo/MainWindow.xaml

```diff
@@ -65,7 +81,7 @@
                     </TextBlock>
                 </StackPanel>
             </DockPanel>
-            <Grid DockPanel.Dock="Top">
+            <Grid Name="grd0" DockPanel.Dock="Top">
```

Comment:
Could we come up with a better name for the grid? 😄

Reply:
Sure. What do you have in mind ?

Reply:
Maybe "SearchGrid"?